smos/src/Smos.hs

```diff
@@ -11,12 +11,13 @@ module Smos
 where
 
 import Brick.BChan as Brick
-import Brick.Main as Brick
+import Brick.Main as BrickS
```

Comment:
I don't understand this change. Is it accidental?

Reply:
Woops. Yes.  CTRL-S going wrong :)